.github/workflows/changesets.yaml

```diff
@@ -129,3 +122,26 @@ jobs:
       - name: Remove tag on failure
         if: failure()
         run: git push --delete origin ${{ env.DASHBOARD_PACKAGE }}@${{ needs.version.outputs.dashboardVersion }}
+
+  publish-vercel:
+    name: Publish to Vercel
+    runs-on: ubuntu-latest
+    needs:
+      - publish-docker
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install Node and dependencies
+        uses: ./.github/actions/install-dependencies
+        with:
+          TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+          TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+      - name: Setup Vercel CLI
+        run: pnpm add -g vercel
+      - name: Trigger a Vercel deployment
+        env:
+          VERCEL_ORG_ID: ${{ secrets.DASHBOARD_VERCEL_TEAM_ID }}
+          VERCEL_PROJECT_ID: ${{ secrets.DASHBOARD_VERCEL_PROJECT_ID }}
+        run: vercel pull --environment=production && vercel build --prod && vercel deploy --prebuilt --prod
```

Comment:
Maybe multiple lines instead of multiple `&&` to improve readability
GH actions have `set -e` anyway

Reply:
> > why would we want to add this step? it is not needed in the next steps, and as it is already executed in a dependent job the turborepo remote cache should already be available when building the images.
> > Yes, why not creating a separate job for Vercel
> 
> Oh, i missed the fact the build+deploy to vercel was happening inside the GH action - thought we were still triggering the hook. Nice

Nope. Vercel runs the build step ignore script even if we trigger the deployment through the webhook. We ignored `main` branch on purpose for our production deployment, but that meant that even webhooks would be ignored, so we need to use the CLI.